### DISCUSSION TITLE
Seems like AutoImport and stuff not working properly

### DISCUSSION THREAD

**Discussion by chrisjung-dev** [antfu/vitesse#224] [Q&A]
Using my private Notebook, LTS Node (14) on Manjaro, VSCode with all recommended Plugins and Stuff.

Auto Imports are generated, Components file is created and (sometimes) updated.

But: Auto import for apis does not work, auto import of my own components do not work. The Auto-Importing of Icon classes does not work (components.d.ts gets no update für icons, even if VSCode shows the correct icon.

What am I missing?

**Comment by antfu** [comment]
Can you share more context?

**Reply by chrisjung-dev** [reply]
I created a repo by using gihubs template function.

Checked it out locally, did the install things, opened VS Code. Built some components. VS Code shows matching info when hovering the component tag.

When I delete components.d.ts and restart Vite server, it gets created and updated with my components. While the server is running, new components won't get added.

Additionally neither auto-imports.d.ts nor components.d.ts get recognized by vite dev server.
Using the default App.vue, this resolved in "useHead" not beeing found an crashing the SPA.

Then I created a new project locally with the box degit, which works ootb.

Removed node_modules, package.lock and pnpm lock file from first repo, Put in the package files from repo 2, ran all the install things. Still not working.

**Reply by chrisjung-dev** [reply]
Same machine, same user, same console, same node version.

**Reply by chrisjung-dev** [reply]
The not working thing is public: https://github.com/campino2k/campino2k.github.io

**Comment by thuongtruong1009** [comment]
Hi @campino2k !
- I think your problem is "put in the package files from repo 2".
- In my opinion, you should put it in 2 separate folders and then install the dependencies to avoid config conflicts, then import it as usual..
- Also, you should also check that router-view and router-link are displayed and set to=" ".
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this.
- Thank you!
